ui/src/components/system/UsersGroups.vue

```diff
@@ -579,19 +589,25 @@
       <div class="modal-dialog">
         <div class="modal-content">
           <div class="modal-header">
-            <h4 class="modal-title">
+            <h4 v-if="newGroup.EditGroup" class="modal-title">
               {{newGroup.isEdit ? $t('edit') : $t('users_groups.create_group')}}
               <span
                 v-if="newGroup.isEdit"
               >{{newGroup.name}}</span>
             </h4>
+            <h4 v-else class="modal-title">
+              {{$t('users_groups.edit_delegation')}}
```

Comment:
```suggestion
              {{$t('users_groups.edit_delegation', {name: newGroup.name})}}
```

For good i18n the strings must use placeholders as the position of "name" (in this case) depends on the language. We cannot assume the correct position of *name* is after the *edit_delegation* label in every possible language. 

A similar rule applies to punctuation: periods and question marks must be in translated strings. 
/cc @edospadoni @gsanchietti 

See also
- https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting
- https://www.gnu.org/software/gettext/manual/html_node/Preparing-Strings.html#Preparing-Strings

Reply:
Put objects, vars in translation is a really bad thing, see what it occurred in French

Reply:
Translators are as programmers: they make errors. Sometimes they don't even know what they are typing :D

Reply:
ok